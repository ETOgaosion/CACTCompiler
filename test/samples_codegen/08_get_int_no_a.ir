============ global var ================
============ imm value ================
============ functions ================
----------- funcDef -----------
main() => 2:
----------- paramVars -----------
----------- localVars -----------
----------- tempVars ------------
t_0:= symbol type: TEMP VAR; data type: 2
----------- Codes ------------
main:
	call get_int; 
	t_0 =  return ;
	addparam t_0;
	call print_int; 
	return 0;

