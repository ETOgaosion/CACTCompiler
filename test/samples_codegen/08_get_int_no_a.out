bbl loader
please enter an int number:
2
